### PR TITLE
Lagom 1.6.1

### PR DESCRIPTION
**Pull request by dwijnand**

### CODE REVIEW

**Review by ignasi35 — APPROVED** [review]

File: src/main/g8/build.sbt [hunk 1]
```diff
@@ -6,6 +6,7 @@ scalaVersion in ThisBuild := "2.13.0"
 
 val macwire = "com.softwaremill.macwire" %% "macros" % "2.3.3" % "provided"
 val scalaTest = "org.scalatest" %% "scalatest" % "3.0.8" % Test
+val scalaXml = "org.scala-lang.modules" %% "scala-xml" % "1.2.0" % Test
```
Comment (ignasi35) [inline src/main/g8/build.sbt]: Wait WAT? Why?

File: project/build.properties [hunk 2]
```diff
@@ -1 +1 @@
-sbt.version=1.2.8
+sbt.version=1.3.8
```
Comment (ignasi35) [inline project/build.properties]: Want to bump `1.3.8` in `lagom/lagom-java.g8` too?

Comment (dwijnand) [reply project/build.properties]: No, feel free.

File: src/main/g8/build.sbt (hunk 1 above)
Comment (dwijnand) [reply src/main/g8/build.sbt]: Trying random things to fix it.  But it isn't a scala-xml thing here.

File: project/build.properties (hunk 2 above)
Comment (ignasi35) [reply project/build.properties]: https://github.com/lagom/lagom-java.g8/pull/77

**Pull request merged**